docs/erc1155-skills.md

```diff
@@ -127,16 +147,15 @@ aea add-key ethereum ethereum_private_key.txt
 
 And one for the P2P connection:
 ``` bash
-aea generate-key cosmos
-aea add-key cosmos cosmos_private_key.txt --connection
+aea generate-key fetchai
+aea add-key fetchai fetchai_private_key.txt --connection
```

Comment:
This means you use the same key in the connection and the aea. That's generally discouraged.

Reply:
what are you suggesting?

Reply:
e.g. `aea generate-key fetchai connection_fetchai_private_key.txt` and `aea add-key fetchai connection_fetchai_private_key.txt --connection` yielding:

```
connection_private_key_paths:
  fetchai: connection_fetchai_private_key.txt
private_key_paths:
  fetchai: fetchai_private_key.txt
```

Reply:
We use `ethereum` as ledger in this demo.

Reply:
same idea